### PR TITLE
Add CONTRIBUTING.md file

### LINKED ISSUE #131: Add CONTRIBUTING.md file
A CONTRIBUTING.md would be a helpful addition to this project for people looking to contribute.

### PR DESCRIPTION
**Pull request by ShrillShrestha**
Resolves #131

### CODE REVIEW

**Review by bodhisha — APPROVED** [review]
Looks cool @ShrillShrestha  🎉

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - Add and commit your changes with `git add <changes>` and `git commit -m "your commit message"`
+  - Then push the changes into your branch `git push origin branch_name`
+  - Before making a PR, make sure your copy of the project is up-to-date with the main project.
+  - If forked project is up-to-date with the main project, go ahead and make the PR.
+  - #### If there has been any commit since you forked the project, there might be some conflicts while making the PR. In case of any conflicts:
+    - Set an upstream with `git remote add upstream https://github.com/MLH-Fellowship/SocioMark.git`
+    - Checkout to main branch and `git pull upstream main`
+    - Checkout to the branch you were working on
+    - Rebase it with the main branch with `git rebase main`
+    - If there is any conflicts while rebasing, resolve them, and continue the rebase
```
Comment (aitikgupta) [inline CONTRIBUTING.md]:
```suggestion
    - If there are any conflicts while rebasing, resolve them locally, and continue the rebase
```

**Review by aitikgupta — CHANGES_REQUESTED** [review]
Just a small grammatical error, other than that it's good to go! 🎉

**Review by DebugAgrawal — APPROVED** [review]
> ### Thanks for Contributing 🚩

**Review by aitikgupta — CHANGES_REQUESTED** [review]
@ShrillShrestha can you also update the README? (It has a Contributing heading -> you can redirect people from there to CONTRIBUTING.md)

**Review by aitikgupta — APPROVED** [review]
Awesome! 🎉

**Pull request merged**